src/main/java/de/hysky/skyblocker/mixins/ClientPlayNetworkHandlerMixin.java

```diff
@@ -148,4 +153,18 @@ public abstract class ClientPlayNetworkHandlerMixin {
     private void skyblocker$onEntityEquip(EntityEquipmentUpdateS2CPacket packet, CallbackInfo ci, @Local Entity entity) {
         EggFinder.checkIfEgg(entity);
     }
+
+    @Inject(method = "onPlayerPositionLook", at = @At("TAIL"))
+    private void onPlayerTeleported(PlayerPositionLookS2CPacket packet, CallbackInfo ci) {
+        //player has been teleported by the server tell the smooth AOTE this
+        SmoothAOTE.reset();
+    }
+
+    @WrapOperation(method = "tick", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/hud/DebugHud;shouldShowPacketSizeAndPingCharts()Z"))
+    private boolean shouldShowPacketSizeAndPingCharts(DebugHud instance, Operation<Boolean> original) {
+        //make the f3+3 screen always send ping packets even when closed
+        //this is needed to make smooth AOTE work
+        return true;
```

Comment:
Maybe only on skyblock? To avoid spamming other servers and using more bandwidth.